res/values/strings.xml

```diff
@@ -346,8 +346,8 @@ there are any).</string>
     <!-- SBB is german/international, CFF is french and FFS is italian! So this IS translatable. -->
     <string name="np_name_sbb" translatable="true">SBB</string>
     <string name="np_desc_sbb">Switzerland</string>
-    <string name="np_desc_vbl">Luzern</string>
-    <string name="np_desc_zvv">Zurich</string>
+    <string name="np_desc_vbl">Lucerne</string>
+    <string name="np_desc_zvv">Zürich</string>
```

Comment:
In the [Wikipedia page](https://en.wikipedia.org/wiki/Z%C3%BCrich) it is used "Zürich" even if it is written that even Zurich is used